Library/Homebrew/extend/ENV/super.rb

```diff
@@ -55,9 +55,17 @@ def reset
       build_bottle:    T.nilable(T::Boolean),
       bottle_arch:     T.nilable(String),
       testing_formula: T::Boolean,
+      debug_symbols:   T.nilable(T::Boolean),
     ).void
   }
-  def setup_build_environment(formula: nil, cc: nil, build_bottle: false, bottle_arch: nil, testing_formula: false)
+  def setup_build_environment(
+    formula: nil,
+    cc: nil,
+    build_bottle: false,
+    bottle_arch: nil,
+    testing_formula: false,
+    debug_symbols: false
+  )
```

Comment:
Keep these in as few lines as rubocop will allow.

Reply:
Done in cd9fe97c551d4e9b5f0a702a1382a6ccbe494773